src/pages/Admin/Admin.tsx

```diff
@@ -102,7 +102,7 @@ const VenueList: React.FC<VenueListProps> = ({
             <Link to={`/admin/venue/${venue.id}`}>{venue.name}</Link>
             {isVenueWithRooms(venue) && (
               <ul className="page-container-adminsidebar-subvenueslist">
-                {venue.rooms.map((room, idx) => (
+                {venue.rooms?.map((room, idx) => (
```

Comment:
This is the issue that was crashing the Admin page. I thought it'd be prudent to fix all the rest of the trouble with the PartyMap template.

Reply:
I suppose our Types are incorrect and give false sense of safety. We should probably have a call and discuss this.

Reply:
This could be an issue in other components as well, at this very moment.

Reply:
https://github.com/sparkletown/sparkle/pull/744 my 'we should fix the types' PR that I keep not having time to get back to.